File: build-system/integration-test/test-projects/lintRelease/app/src/main/java/com/android/test/lint/MainActivity.java
package com.android.test.lint;

import android.os.Bundle;
import com.android.test.applib_library03.LintMainActivity;

public class MainActivity extends LintMainActivity implements Runnable {
    public void run() { }

    public MainActivity() {
        super(false, false);
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }
}

File: build-system/integration-test/test-projects/composeHelloWorld/app/src/main/java/com/example/helloworldcompose/MainActivity.kt
package com.example.helloworldcompose

import android.os.Bundle
import androidx.activity.compose.setContent
import androidx.appcompat.app.AppCompatActivity
import androidx.compose.foundation.layout.padding
import androidx.compose.foundation.layout.Column
import androidx.compose.material.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.tooling.preview.Preview
import androidx.compose.ui.unit.dp


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            HelloWorldColumn()
        }
    }
}

@Composable
fun HelloWorldColumn() {
    Column(modifier = Modifier.padding(16.dp)) {
        Text("Hello, World1")
        Text("Hello, World2")
        Text("Hello, World3")
    }
}

@Preview
@Composable
fun PreviewGreeting() {
    HelloWorldColumn()
}
